aten/src/ATen/native/LegacyDefinitions.cpp

```diff
@@ -54,6 +54,11 @@ Tensor argsort(const Tensor & self, int64_t dim, bool descending) {
   return std::get<1>(at::sort(self, dim, descending));
 }
 
+Tensor & cauchy_cuda_(Tensor& self, double median, double sigma, Generator * generator) {
+  return at::legacy::th::_th_cauchy_(self, median, sigma, generator);
```

Comment:
Remove this function when this is in: https://github.com/pytorch/pytorch/pull/20622